docker-compose.yml

```diff
@@ -7,32 +7,47 @@ services:
       dockerfile: Dockerfile
     image: eventq
     container_name: testrunner
-    command: bash -c "./scripts/container_loop.sh"
+    command: bash -c "./script/container_loop.sh"
```

Comment:
Was this never working?

Reply:
It was, we just readjusted some paths:

@florianpilz
[Adjust script and source path](https://github.com/Sage/eventq/pull/106/commits/9f5c0c4d5f19b9cb0b7af8055208239305b8a761)
[9f5c0c4](https://github.com/Sage/eventq/pull/106/commits/9f5c0c4d5f19b9cb0b7af8055208239305b8a761)

As we are now setting `src` as the default folder, we can just use the
container loop from the mounted `src` volume. Also adjust paths in
README and test script to work from root, rather inside the `script`
folder.